app/assets/javascripts/mapboxgl.coffee

```diff
@@ -39,12 +39,7 @@ window.initialize_mapboxgl = (elmID) ->
 
 get_map_loader = (map) ->
   map_id = map.getContainer().id
-
-  if map_id == 'all_results_map'
-    loader_id = '#loader'
-  else if map_id == 'zip_code_map'
-    loader_id = '#stats_loader'
-
+  loader_id = '#loader'
```

Comment:
Is this in an effort to move all stats to their own page?

Reply:
No, there just isn't a 2nd map and the branch above wasn't needed anymore.